docs/user/orion.rst

```diff
@@ -0,0 +1,92 @@
+.. _orion:
```

Comment:
In general, I suggest going through the documentation here and making the usage of a full stop at the end of a sentence consistent.